packages/core/src/textures/BaseTexture.ts

```diff
@@ -704,11 +704,11 @@ export class BaseTexture<R extends Resource = Resource, RO = IAutoDetectOptions>
      * @returns - The resulting new BaseTexture
      */
     static fromBuffer(buffer: BufferType, width: number, height: number,
-        options?: IBaseTextureOptions): BaseTexture<BufferResource>
+        options?: IBaseTextureOptions<IBufferResourceOptions>): BaseTexture<BufferResource>
     {
         buffer = buffer || new Float32Array(width * height * 4);
 
-        const resource = new BufferResource(buffer, { width, height });
+        const resource = new BufferResource(buffer, Object.assign({ width, height }, options?.resourceOptions));
```

Comment:
```suggestion
        const resource = new BufferResource(buffer, { width, height, ...options?.resourceOptions });
```

Maybe it's more concise to write it this way?